diffs/packetbeatdiff.diff

```diff
@@ -0,0 +1,2602 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_compression_level_literal">
++                <a id="_compression_level">
+                 </a>
+                 <code class="literal">
+                  compression_level
+@@ -444,7 +444,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_escape_html_literal">
++                <a id="_escape_html">
+                 </a>
+                 <code class="literal">
+                  escape_html
+@@ -476,7 +476,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_worker_literal">
++                <a id="_worker">
+                 </a>
+                 <code class="literal">
+                  worker
+@@ -500,7 +500,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_username_literal">
++                <a id="_username">
+                 </a>
+                 <code class="literal">
+                  username
+@@ -521,7 +521,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_password_literal">
++                <a id="_password">
+                 </a>
+                 <code class="literal">
+                  password
+@@ -542,7 +542,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_parameters_literal">
++                <a id="_parameters">
+                 </a>
+                 <code class="literal">
+                  parameters
+@@ -627,7 +627,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_headers_literal">
++                <a id="_headers">
+                 </a>
+                 <code class="literal">
+                  headers
+@@ -655,7 +655,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_proxy_url_literal">
++                <a id="_proxy_url">
+                 </a>
+                 <code class="literal">
+                  proxy_url
+@@ -695,7 +695,7 @@
+             <p>
+              The index name to write events to when you’re using daily indices. The default is
+              <code class="literal">
+-              "packetbeat-%{[agent.version]}-%{+yyyy.MM.dd}"
++              "packetbeat-%\{[agent.version]\}-%\{+yyyy.MM.dd\}"
+              </code>
+              (for example,
+              <code class="literal">
+@@ -753,7 +753,7 @@
+             <div class="pre_wrapper lang-yaml">
+              <pre class="programlisting prettyprint lang-yaml">output.elasticsearch:
+   hosts: ["http://localhost:9200"]
+-  index: "%{[fields.log_type]}-%{[agent.version]}-%{+yyyy.MM.dd}" <a id="CO9-1"></a><i class="conum" data-value="1"></i></pre>

```

Comment:
fixed in https://github.com/elastic/beats/pull/13722/commits/def5e8ef969937918badbd5ddf624addb3f3536c